### DISCUSSION TITLE
Q: Zeabur部署初始化失败 || Q: Zeabur deployment failed to initialize

### DISCUSSION THREAD

**Discussion by AkimioJR** [walinejs/waline#2107] [Q&A]
### Describe the bug | 问题描述

使用Zeabur部署Waline，数据库使用Vercel自带的免费postgres，注册账户时初始化失败（已初始化三个表）
![image](https://github.com/walinejs/waline/assets/77880495/93463c8f-bc51-4123-9c9e-d36a04267429)
相同的环境变量在Vercel上就成功部署

### Website URL | 问题网站

https://dqwdqw.zeabur.app/

### Where your waline deploy? | 服务部署在哪里？

Zeabur

### Where your comment data store? | 数据存储在哪里？

PostgreSQL

---
### Describe the bug | Problem description

Use Zeabur to deploy Waline. The database uses the free postgres that comes with Vercel. The initialization failed when registering an account (three tables have been initialized).
![image](https://github.com/walinejs/waline/assets/77880495/93463c8f-bc51-4123-9c9e-d36a04267429)
The same environment variables are successfully deployed on Vercel

### Website URL | Problem website

https://dqwdqw.zeabur.app/

### Where your waline deploy? | Where is the service deployed?

Zeabur

### Where your comment data store? | Where is the data stored?

PostgreSQL

**Comment by lizheming** [comment]
需要添加一下 pgsql 相关的环境变量，默认脚手架只适配了 mongodb 的 zeabur 变量~

---
You need to add pgsql related environment variables. The default scaffolding is only adapted to mongodb’s zeabur variable~

**Comment by AkimioJR** [comment]
> 需要添加一下 pgsql 相关的环境变量，默认脚手架只适配了 mongodb 的 zeabur 变量~
> 
> You need to add pgsql related environment variables. The default scaffolding is only adapted to mongodb’s zeabur variable~

可以使用MongoDB官方的数据库而不是Zeabur自带的MongoDB数据库吗

**Reply by lizheming** [reply]
可以，自己配置环境变量即可

---
Yes, you can configure the environment variables yourself
